resume-buddy/src/main/webapp/resume-review-script.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
JavaScript indentations should be 2 spaces (https://google.github.io/styleguide/jsguide.html#formatting-block-indentation). Is this being auto-formatted to 6 spaces by Cloud Shell Editor? :(

Reply:
Yep, forgot to change tab size for JavaScript files. Should be fixed now